docs/Governance.md

```diff
@@ -115,6 +115,17 @@ Responsibilities:
 * Manage github settings such as branch protection rules and repository secrets.
 * Resolve any disputes among maintainers.
 
+## Prioritizing Issues
+
+In the weekly triage meetings, new issues are assigned to contributors and are set to a monthly milestone by which the issue is expected to be completed. It may not be possible for the assignees to address all the issues assigned to them by their target milestone. As a result issues may pile up in a given milestone. In such cases project admins can request contributors to use *special labels* to prioritize issues as follows:
+### P1:
+This label represents highest priority. This label should be used for bugs impacting users, new feature or enhancements that are urgently requested by users. Contributors should try to resolve these issues in the **current milestone**.
```

Comment:
As lines 28-29 explain, it's Collaborators (not Contributors) who can have issues assigned to them.  Hence I think this should be:
```suggestion
This label represents highest priority. This label should be used for bugs impacting users, new feature or enhancements that are urgently requested by users. Collaborators should try to resolve these issues in the **current milestone**.
```